app/utils/discogs.schemas.ts

```diff
@@ -1,16 +1,29 @@
 import z from "zod";
 
-const AlbumSchema = z.object({
+const DiscogsAlbumSchema = z.object({
```

Comment:
this doesn't need to be separate; I would just inline it in `DiscogsResponseSchema`